core/src/game_state.rs

```diff
@@ -1452,10 +1476,26 @@ impl DrawPhase {
                         } else if new_bid.count == existing_bid.count {
                             match (new_bid.card, existing_bid.card) {
                                 (Card::BigJoker, Card::BigJoker) => (),
-                                (Card::BigJoker, _) => valid_bids.push(new_bid),
+                                (Card::BigJoker, _) => {
```

Comment:
you can do

```
(Card::BigJoker, _) if self.propagated.joker_overbid_selection_policy == JokerOverBidSelectionPolicy::AllowEqualOrGreaterLength => {

}
```

to reduce the nesting depth here
```

Reply:
added by cargo fmt, ci will fail without the extra brackets